src/is/types.js

```diff
@@ -125,6 +125,16 @@ export function isWindowObject(value) {
     return value != null && typeof value === 'object' && 'setInterval' in value;
 }
 
+/**
```

Comment:
This should be done in this file, but in `regexp.js`as mentioned in the issue.
Check out @boristane pull request #40